templates/meta_woz/dialogues/templates.yaml

```diff
@@ -0,0 +1,111 @@
+dataset: meta_woz
+subset: dialogues
+templates:
+  4b64c6e9-0aa0-431f-85b5-8367daa4773a: !Template
+    id: 4b64c6e9-0aa0-431f-85b5-8367daa4773a
+    jinja: "What does this conversation between a chatbot and a client talk about\
+      \ ? \n{{turns | join(',  ')}}\n|||\n{{domain.replace('_', ' ') | lower }}"
```

Comment:
The "domains" listed in this dataset are superintuitive to me. Can you provide the list of possible domains to the model in the prompt?

Reply:
Sorry, "super counterintuitive"

Reply:
I thought the same, however the list of domains is big, it seems 51 https://huggingface.co/datasets/meta_woz#data-splits

Reply:
I see, yes, that seems like it is probably too big.

Reply:
I think it's probably better to join the turns with a newline - it's possible that ", " could appear in a turn, making it ambiguous.